usagov/usagov-2021
# USAGOV-1850-wizard-skip-nav-language: Ensure the lang variable gets set on taxonomy term pages in html.html.twig.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1850

## Description

Add twig logic to html.html.twig to set lang variable on taxonomy term pages.

## Type of Changes

- [ X ] Bugfix
  - Add screenshot showing what it should look like

## Testing Instructions


1. Visit any English node page and scroll to the bottom.
2. The "back to top" button appears expected (this is just regression testing).
3. Open devtools and ctrl + f for the`<a>`tag with the href value "#skip-to-h1". 
4. The text "Skip to main content" appears in this tag.
5. Check the node in spanish and find the same tag.
6. The text "Saltar al contenido principal" appears in the tag.
7. Repeat the steps with taxonomy term pages.
8. The language in the skip-to-h1 tag appears as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
9. Find the commit of this pull request.
10. Build and deploy the changes.
11. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
